test/cql-pytest/test_tablets.py

```diff
@@ -46,6 +48,18 @@ def test_create_loop_with_tablets(cql, test_keyspace_128_tablets):
         cql.execute(f"CREATE TABLE {table} (p int PRIMARY KEY, v int)")
         cql.execute("DROP TABLE " + table)
 
+@pytest.mark.parametrize(
+    "strategy_name, dc, expectation",
+    [('SimpleStrategy', 'replication_factor', pytest.raises(InvalidRequest, match="\"SimpleStrategy\" is not supported with tablets")),
+     ('NetworkTopologyStrategy', pytest.lazy_fixture('this_dc'), does_not_raise()),
```

Comment:
I think the `dc` parameter is unnecessary. `'replication_factor'` should also work for NetworkTopologyStrategy: https://enterprise.docs.scylladb.com/stable/cql/ddl.html#networktopologystrategy.

Reply:
ALTER tablets KS rejects 'replication_factor' tag. Take a look at [6.0 OSS documentation](https://opensource.docs.scylladb.com/stable/cql/ddl.html#alter-keyspace-with-tablets-experimental) (the one you linked refers to the enterprise):
> The ALTER statement rejects the replication_factor tag. List the DCs explicitly when altering a keyspace. See [NetworkTopologyStrategy](https://opensource.docs.scylladb.com/stable/cql/ddl.html#replication-strategy)

Reply:
That's ALTER KS, but CREATE KS in the test should work. Anyway, it's just a detail.

Reply:
Oh, sorry, you're right! I'll fix it, one parameter less will make the test more readable, I think. And I'm surprised it's only disabled for ALTER, but OK, it is what it is.